src/btree/row_modify.c

```diff
@@ -328,16 +328,12 @@ __wt_update_obsolete_check(
     WT_TXN_GLOBAL *txn_global;
     WT_UPDATE *first, *next;
     size_t size;
-    uint64_t oldest, stable;
-    u_int count, upd_seen, upd_unstable;
+    u_int count;
 
     next = NULL;
     page = cbt->ref->page;
     txn_global = &S2C(session)->txn_global;
 
-    upd_seen = upd_unstable = 0;
```

Comment:
As there is a use for these variables in this function, instead of removing the code, just keep it.

Reply:
Placed back [7a7da52](https://github.com/wiredtiger/wiredtiger/pull/9690/commits/7a7da52b5c5b83396135abcb3fb9bca17cd91dc8)